### DISCUSSION TITLE
the test does not run

### DISCUSSION THREAD

**Discussion by snippet2021** [yasserfarouk/scml#93] [General]
I submitted my OneShot program from the home page and clicked on "Test Active Agent". 
However, the test does not run. 
My Agent ID is 20097.Please check the status.

**Comment by yasserfarouk** [comment]
Thank you for your submission. We could reproduce the issue and we are working to resolve it. I will inform you once it is fully resolved.

**Reply by snippet2021** [reply]
Thank you!

**Reply by yasserfarouk** [reply]
@snippet2021  we think the problem is resolved now. Please try to upload your agent again.

**Reply by snippet2021** [reply]
@yasserfarouk 
I was able to pass the test now. 
Thank you for your support.

**Reply by yasserfarouk** [reply]
@snippet2021 Great!!
